CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ All notable changes to this project will be documented in this file.
 - Add `Ms-Oc-User-Agent` as default request headers
 - Log `RequestHeaders`
 
+- Added axios-retry library support
```

Comment:
Changelog should be under `[Unreleased]` since we already released 0.5.1 and 0.5.0

Reply:
updated changelog.md file